pkg/kots/lint.go

```diff
@@ -344,6 +361,104 @@ func lintWithKubevalSchema(renderedFiles SpecFiles, originalFiles SpecFiles, sch
 	return lintExpressions, nil
 }
 
+func checkIfKotsVersionExists(version string) (bool, error) {
+	url := "http://api.github.com/repos/replicatedhq/kots/commits/%s"
+	token := os.Getenv("GITHUB_API_TOKEN")
+	var bearer = "Bearer " + token
+
+	if !strings.HasPrefix(version, "v") {
+		version = "v" + version
+	}
+
+	rwMutex.RLock()
+	verIsCached := kotsVersions[version]
+	rwMutex.RUnlock()
+
+	if !verIsCached {
+		req, err := http.NewRequest("GET", fmt.Sprintf(url, version), nil)
+		if err != nil {
+			return false, errors.Wrap(err, "failed to create new request")
+		}
+		req.Header.Set("Authorization", bearer)
+		client := &http.Client{}
+		resp, err := client.Do(req)
+		if resp.StatusCode == 422 {
```

Comment:
interesting, why 422? i've tried hitting the api directly using curl and i'm getting 404
```bash
$ curl -I https://api.github.com/repos/replicatedhq/kots/releases/tags/v1.70.0
HTTP/2 404
```

Reply:
oh i see you're using the `/commits` endpoint instead of the `/tags/` one. let's change that to use the tags one instead.